### ISSUE TITLE
Downloaded ddtall-win-installer-23.06.13-python-3.8.6-32bits.exe but opens up version 2.1.1

### ISSUE THREAD

**Issue by AndyIreland** [cedricp/ddt4all#934]
I downloaded version 3.8.6 from the release area - I chose to place a link on desktop to start, I click on desktop icon to start and it starts up version 2.1.1 of DDT4ALL what have I done wrong?
I am working with win32 - i dont want to use the 64bit version 

![image](https://github.com/cedricp/ddt4all/assets/49172217/81fac611-ac99-4917-88ea-bc1050c081a9)

Thanks

**Comment by KarelSvo** [comment]
3.8.6 python version
2.1.1 ddt4all version

**Comment by AndyIreland** [comment]
This is what target my desktop icon opens up:
"C:\Program Files (x86)\ddt4all\Python38-32\python.exe" "C:\Program Files (x86)\ddt4all\main.py"

**Comment by KarelSvo** [comment]
And where is the problem now?

**Comment by AndyIreland** [comment]
> And where is the problem now? C:\Program Files(x86)\ddtall\Python38-32/NEWS

I click on the desktop icon and it starts version 2.1.1 :
![image](https://github.com/cedricp/ddt4all/assets/49172217/d816971e-2b3d-43ae-ab88-981e49d7df97)

**Comment by KarelSvo** [comment]
Yes, that is correct, it is determined by the version.py file https://github.com/cedricp/ddt4all/blob/v2.1.1/version.py
And now the problem?

**Comment by AndyIreland** [comment]
I dont get it. If I have installed ddtall-win-installer-23.06.13-python-3.8.6-32bits.exe ... and it places a desktop shortcut when installed , and I press on the icon on the desktop why is it opening up the version 2.1.1 DDT4ALL and not opening up a version of 3.8.6 DDT4ALL?
I cannot see how I can explain it any more clearer

**Comment by KarelSvo** [comment]
Because 3.8.6 is the version of the Python programming language and not the version of the ddt4all software.

ddtall-win-installer  **software name**
-23.06.13 **date** 
-python-3.8.6- **python version**
32bits.exe  **windows version**

**Comment by AlMeu** [comment]
@AndyIreland you give advices at many here, but you don't make difference between pyton version and ddt4all version?

**Comment by KarelSvo** [comment]
[This](https://github.com/cedricp/ddt4all/discussions/930#discussioncomment-7490140)   is no longer a problem because no reaction from you?

**Comment by AndyIreland** [comment]
> 3.8.6 is the latest possible version for Windows 7. https://www.python.org/downloads/windows/

ah right that makes sense now - I thought maybe there was a newer version of DDT4ALL called version 3.8.6 
sorry I was confused

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/discussions/933#discussioncomment-7541397
